daemon/config/config.go

```diff
@@ -415,6 +415,8 @@ func MergeDaemonConfigurations(flagsConfig *Config, flags *pflag.FlagSet, config
 		return nil, err
 	}
 
+	PopulateFeatures(fileConfig.Features)
```

Comment:
Wondering if this is the right place to add these; is this something we could add to `setPlatformDefaults`
https://github.com/moby/moby/blob/4d2fbe325aa0e5a2279d5993621bfa64aabdb902/daemon/config/config_windows.go#L79
https://github.com/moby/moby/blob/4d2fbe325aa0e5a2279d5993621bfa64aabdb902/daemon/config/config_linux.go#L221

So that `config.New` returns the config including the defaults? https://github.com/moby/moby/blob/4d2fbe325aa0e5a2279d5993621bfa64aabdb902/daemon/config/config.go#L310-L311